core/src/avm2/globals/flash/globalization/CurrencyFormatter.as

```diff
@@ -8,5 +9,10 @@ package flash.globalization {
         public function CurrencyFormatter(locale: String) {
             stub_constructor("flash.globalization.CurrencyFormatter");
         }
+
+        public function format(value:Number, withCurrencySymbol:Boolean = false):String {
+            return value.toString();
+            stub_method("flash.globalization.CurrencyFormatter", "format");
```

Comment:
Shouldn't this come before the `return` statement...? In other stubbed functions they are in that order.